argus/config/cloudbaseinit.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2016 Cloudbase Solutions Srl
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+"""Config options available for the cloudbase-init setup."""
+
+from oslo_config import cfg
+
+from argus.config import base as conf_base
+
+
+class CloudbaseInitOptions(conf_base.Options):
+
+    """Config options available for the cloudbase-init setup."""
+
+    def __init__(self, config):
+        super(CloudbaseInitOptions, self).__init__(config, group="argus")
+        self._options = [
+            cfg.StrOpt("created_user", default=None),
```

Comment:
The defaults were `Admin` and group `Administrators`, but it's not a deal breaker.